### ISSUE TITLE
Missing strings for translation

### ISSUE THREAD

**Issue by hide-me** [jgraph/drawio#4006]
### Preflight Checklist

- [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

If you are technical, you should reporting bugs along the lines of https://marker.io/blog/how-to-write-bug-report. If you are not technical, we will make allowances, please try to make an effort to understand the process.

**Describe the bug**
Certain strings are missing in the translation table, leading to some interface elements not being translated. The issue includes screenshots for reference.

**To Reproduce**
1. Go to draw.io
2. Switch between Russian and English
3. See missing strings

**Expected behavior**
All interface elements should be appropriately translated.

**Screenshots**
![ru1](https://github.com/jgraph/drawio/assets/45032974/b44a12a3-b43f-457c-8ea2-e83d1bd85f0f)
![ru2](https://github.com/jgraph/drawio/assets/45032974/7920d3fb-8a5e-4cfe-88f7-dd16f0b3a776)
![ru3](https://github.com/jgraph/drawio/assets/45032974/17ac9930-b321-4f18-aea0-d40286fecb09)
![ru4](https://github.com/jgraph/drawio/assets/45032974/a302e298-51cb-49ee-a29b-e2222f70db75)
![ru5](https://github.com/jgraph/drawio/assets/45032974/0bfec807-893f-4060-aca1-e1ed7fc578a2)
To easily find these UI parts I duplicated sceenshots in English
![en1](https://github.com/jgraph/drawio/assets/45032974/dd4b730b-04d3-4ee5-84ba-3353655bbe40)
![en2](https://github.com/jgraph/drawio/assets/45032974/bff570b1-f7e4-41a1-a5bc-7ba99a822ba8)

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 22.1.4

**Additional context**
Probably you know another parts of UI with missing strings -- so, don't be shy and add more screenshots below.

**Comment by davidjgraph** [comment]
https://docs.google.com/spreadsheets/d/1FoYdyEraEQuWofzbYCDPKN7EdKgS_2ZrsDrOA8scgwQ

**Comment by hide-me** [comment]
@davidjgraph 
Ok, I know how translation process looks in draw.io
Because I translated into Russian

But, idk what I should to add in first column, and how to find the person who may help me with it.

**Comment by davidjgraph** [comment]
Ah OK, you're saying these strings are not localised at all? We'll deal with adding them.

**Comment by hide-me** [comment]
Yes Sir, someone who know how to operate with localization (adding new strings for translation, NOT TRANSLATION into a language) should add missing items. It would be amazing.

And then, I will try to translate it to Russian 😄 

Cheers, @davidjgraph!
